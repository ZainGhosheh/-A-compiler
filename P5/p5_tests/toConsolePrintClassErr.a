m : custom {
};

f : () -> void {
	toconsole m;
}
