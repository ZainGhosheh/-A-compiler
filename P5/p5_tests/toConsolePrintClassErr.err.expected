FATAL [5,2]-[5,13]: Attempt to output a class
Type Analysis Failed
